platform/mds/mds/src/main/java/org/motechproject/mds/domain/Lookup.java

```diff
@@ -155,9 +160,13 @@ public LookupDto toDto() {
 
             String customOperator = getCustomOperators().get(lookupFieldName);
             boolean useGenericParam = toBoolean(getUseGenericParams().get(lookupFieldName));
-            LookupFieldDto lookupField = new  LookupFieldDto(field.getId(), field.getName(), lookupFieldType, customOperator,
-                    useGenericParam, LookupName.getRelatedFieldName(lookupFieldName));
-            lookupFields.add(lookupField);
+            if (field != null) {
+                LookupFieldDto lookupField = new LookupFieldDto(field.getId(), field.getName(), lookupFieldType, customOperator,
+                        useGenericParam, LookupName.getRelatedFieldName(lookupFieldName));
+                lookupFields.add(lookupField);
+            } else {
+                LOGGER.info("Can't create LookupFieldDto from field with name {}", lookupFieldName);
```

Comment:
I believe we should rather from an exception in this case - with the correct message. Proceeding should not be possible in this case.